Binance Detect Moonings.py

```diff
@@ -54,39 +55,53 @@
 PAIR_WITH = 'USDT'
 
 # Define the size of each trade, by default in USDT
-QUANTITY = 100
+QUANTITY = 15
+
+# Define max numbers of coins to hold
+MAX_COINS = 5
```

Comment:
can this be set to 0 for "unlimited"?

Reply:
-1 will do the trick.

Reply:
If you change to the following it will :-)
if len(coins_bought) < MAX_COINS or MAX_COINS == 0: